### PR TITLE
Abby ucsb org table

### PR DESCRIPTION
**Pull request by Abby-Fan04**

### CODE REVIEW

**Review by yungongWang — COMMENTED** [review]
If you want to make this change, I am not sure if we need to change the content in UCSBDiningCommons.json

File: src/main/resources/db/migration/changes/UCSBOrganizations [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          "author": "Abby",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "UCSBORGANIZARIONS"
```
Comment (yixuan-wong) [inline src/main/resources/db/migration/changes/UCSBOrganizations]: Spelling error?

File: src/main/resources/db/migration/changes/UCSBOrganizations [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    "column": {
+                      "constraints": {
+                        "nullable": false
+                      },
+                      "name": "INACTIVE",
+                      "type": "BOOLEAN"
+                    }
+                  }]
+                ,
+                "tableName": "UCSBORGANIZARIONS"
```
Comment (yixuan-wong) [inline src/main/resources/db/migration/changes/UCSBOrganizations]: Spelling error?

**Review by yixuan-wong — CHANGES_REQUESTED** [review]
I think UCSBDiningCommons.json should not be touched so the edit on line 14 should be reverted. Also, are there spelling mistakes in your database migration file? Specifcally "tableName" on lines 15 and 57.

**Review by yixuan-wong — APPROVED** [review]
LGTM

**Pull request merged**